packages/zoe/test/types.test-d.ts

```diff
@@ -20,13 +20,13 @@ import type { prepare as scaledPriceAuthorityStart } from '../src/contracts/scal
   const kit = await E(zoe).startInstance(scaledPriceInstallation);
   // @ts-expect-error
   kit.notInKit;
-  E(kit.publicFacet).getPriceAuthority();
+  void E(kit.publicFacet).getPriceAuthority();
```

Comment:
There are driveby fixes, yes? Meaning, they aren't related to the purpose of this PR but a fix made in passing and bundled in. I'm all in favor; just making sure I understand.

Reply:
Yep, separate commit [6dfc8e3](https://github.com/Agoric/agoric-sdk/pull/9092/commits/6dfc8e326604bc8325b4205453eb868a81825168)